mailbox/api/src/main/java/org/apache/james/mailbox/model/UpdatedFlags.java

```diff
@@ -146,16 +146,16 @@ public Flags getOldFlags() {
         return oldFlags;
     }
 
-    public boolean isSet(Flags.Flag flag) {
+    public boolean wasSet(Flags.Flag flag) {
```

Comment:
that's worst for me: wasSet should mean that only oldFlag is concerned, and that oldFlag contains it.
maybe isModifiedToSet? (don't find better, sorry)
and this other should be then isModifiedToUnset and isUnchanged.